### ISSUE TITLE
Django 2.0 support

### ISSUE THREAD

**Issue by barseghyanartur** [barseghyanartur/django-fobi#107]
Django 2.0 support.

Work in progress in [branch master](https://github.com/barseghyanartur/django-fobi/tree/master).

All tests pass, however, it's too early to proclaim that Django 2.0 is supported.

Thus:

- **Needs to be tested more**
- **Help appreciated**

The requirements used (Python 3.6):

```text
alabaster==0.7.6
aspy.yaml==1.0.0
astroid==1.6.0
Babel==2.1.1
cached-property==1.3.1
certifi==2017.11.5
chardet==3.0.4
coverage==4.4.2
decorator==4.0.4
Django==2.0
django-admin-tools==0.8.1
django-autoslug-iplweb==1.9.4.dev0
django-ckeditor==5.4.0
django-debug-toolbar==1.6
django-formtools==2.1
django-js-asset==0.1.1
django-nine==0.1.13
django-nonefield==0.2
django-ranged-response==0.2.0
django-registration-redux==2.0
django-simple-captcha==0.5.6
djangorestframework==3.7.7
docopt==0.4.0
docutils==0.12
easy-thumbnails==2.4.1
factory-boy==2.7.0
fake-factory==0.7.2
Faker==0.7.11
flake8==3.5.0
identify==1.0.7
idna==2.6
ipdb==0.10.3
ipython==6.2.1
ipython-genutils==0.2.0
isort==4.2.15
jedi==0.11.1
Jinja2==2.8
lazy-object-proxy==1.3.1
mailchimp==2.0.9
Markdown==2.6.10
MarkupSafe==0.23
mccabe==0.6.1
nodeenv==1.2.0
olefile==0.44
ordereddict==1.1
parso==0.1.1
path.py==8.1.2
pexpect==4.0.1
pickleshare==0.5
Pillow==4.2.1
pluggy==0.3.1
pre-commit==1.4.1
prompt-toolkit==1.0.15
psycopg2==2.7.3.1
ptyprocess==0.5
py==1.5.2
pycodestyle==2.3.1
pydocstyle==2.1.1
pyflakes==1.6.0
Pygments==2.0.2
pylint==1.8.1
pytest==3.0.2
pytest-cov==2.2.1
pytest-django==2.9.1
python-dateutil==2.6.1
pytz==2015.6
PyYAML==3.12
requests==2.18.4
selenium==2.53.6
simplegeneric==0.8.1
simplejson==3.8.0
six==1.10.0
snowballstemmer==1.2.0
Sphinx==1.3.1
sphinx-rtd-theme==0.1.9
sqlparse==0.2.2
tox==2.1.1
traitlets==4.3.2
Unidecode==0.4.21
urllib3==1.22
virtualenv==13.1.2
vishap==0.1.5
wcwidth==0.1.7
wheel==0.24.0
wrapt==1.10.11
```

**Comment by barseghyanartur** [comment]
Work in progress, [branch django20](https://github.com/barseghyanartur/django-fobi/tree/django20).

**Comment by barseghyanartur** [comment]
Implemented in 0.12.10.
